underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -175,6 +187,9 @@ public boolean delete(String path, boolean recursive) throws IOException {
   @Override
   public boolean exists(String path) throws IOException {
     String newPath = stripPrefixIfPresent(path);
+    if (newPath.endsWith("_temporary")) {
+      return true;
+    }
```

Comment:
There are already several fixes going into this PR, could we extract this one into another PR?